xiblint/__main__.py

```
@@ -58,22 +58,24 @@ def main():
     for path in include_paths:
         if os.path.isfile(path):
             checkers = config.checkers(path)
-            errors += process_file(path, checkers, args.reporter)
-        else:
+            errors += process_file(path, checkers)
+        elif os.path.isdir(path):
             for root, _, files in os.walk(path):
-                for file_path in [os.path.join(root, file) for file in files]:
-                    checkers = config.checkers(file_path)
-                    errors += process_file(file_path, checkers, args.reporter)
+                for filename in files:
+                    if os.path.splitext(filename)[1].lower() in ('.storyboard', '.xib'):
+                        file_path = os.path.join(root, filename)
+                        checkers = config.checkers(file_path)
+                        errors += process_file(file_path, checkers)
+        else:
+            print("Error: Invalid path '{}'".format(path))
```

Now we won't silently error out if path does not exist (or doesn't end in .storyboard / .xib).

---

Awesome 👍